blog-homepage/src/components/YourArticles/YourArticles.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Media from 'react-bootstrap/Media';
+import '../Articles.css'
+import {faStar, faVolumeUp} from '@fortawesome/free-solid-svg-icons';
+import AuthorInfo from '../AuthorInfo/AuthorInfo';
+import PropTypes from 'prop-types';
+
+const YourArticles = props => {
+  const {image, hasAudioAvailable, memberPreview, title, description, link } = {...props.data};
```

Comment:
```suggestion
  const {image, hasAudioAvailable, memberPreview, title, description, link } = props.data
```
`{...props.data}` is using the ES6 spread operator to take all of props.data object's keys and wrap them in a new object. `props.data = {...props.data}`